tests/integration/test_writes/test_writes.py

```diff
@@ -916,12 +916,28 @@ def test_sanitize_character_partitioned(catalog: Catalog) -> None:
 
 
 @pytest.mark.parametrize("format_version", [1, 2])
-def table_write_subset_of_schema(session_catalog: Catalog, arrow_table_with_null: pa.Table, format_version: int) -> None:
-    identifier = "default.table_append_subset_of_schema"
+def test_table_write_subset_of_schema(session_catalog: Catalog, arrow_table_with_null: pa.Table, format_version: int) -> None:
```

Comment:
I noticed this as well while working on https://github.com/apache/iceberg-python/pull/848 :)

Should we also mark these tests as integration tests by tagging it with `@pytest.mark.integration` ?